src/main/java/com/redhat/cloud/policies/app/PolicyEngine.java

```diff
@@ -126,6 +126,12 @@ Response findLastTriggered(@QueryParam("triggerIds") String triggerIds,
                                 @QueryParam("thin") boolean thin,
                                 @QueryParam("page") int page,
                                 @QueryParam("per_page") int per_page,
+                                // Once MP-RestClient 2.0 is avaliable, we can turn the next lines
+                                // into String[] and set the query param style accordingly.
+                                @QueryParam("sort") String sort1,
+                                @QueryParam("sort") String sort2,
+                                @QueryParam("order") String order1,
+                                @QueryParam("order") String order2,
```

Comment:
Is this for sorting by 2 columns?
I don't think the current UI components support that out out of the box.

Reply:
Ah, but i see that it could sort using `ctime` by default on the second column.

Reply:
If we can only sort by one column, I am happy too :)